### PR TITLE
Security update sfcc node version

### PR DESCRIPTION
**Pull request by joyet-simon**
## Reason for change

[Linear task](https://linear.app/almapay/issue/MPP-xxx/)

### How to test

_As a reviewer, you are encouraged to test the PR locally._

### Checklist for authors and reviewers

- [ ] The title of the PR uses business wording, not technical jargon, for the changelog readers to understand it
- [ ] The PR implements the changes asked in the referenced task / issue
- [ ] The automated tests are compliant with the [testing strategy](https://www.notion.so/almapay/Backend-testing-strategy-06c642cec1bf47b9b8feca3a91ea8d4a)
- [ ] The tests are relevant, and cover the corner/error cases, not only the happy path
- [ ] You understand the impact of this PR on existing code/features
- [ ] The changes include adequate logging and Datadog traces
- [ ] Documentation is updated (API, developer documentation, ADR, Notion...)

### CODE REVIEW

File: .github/workflows/aqua.yml [hunk 1]
```diff
@@ -7,25 +8,43 @@ on:
...
         env:
           AQUA_KEY: ${{ secrets.AQUA_KEY }}
           AQUA_SECRET: ${{ secrets.AQUA_SECRET }}
           GITHUB_TOKEN: ${{ github.token }}
           AQUA_URL: https://api.eu-1.supply-chain.cloud.aquasec.com
           CSPM_URL: https://eu-1.api.cloudsploit.com
-          TRIVY_RUN_AS_PLUGIN: "aqua"
-          # For http/https proxy configuration add env vars: HTTP_PROXY/HTTPS_PROXY, CA-CRET (path to CA certificate)
+          TRIVY_RUN_AS_PLUGIN: aqua
+          TRIVY_DB_REPOSITORY: europe-docker.pkg.dev/lyrical-carver-335213/ghcr-remote-cache/aquasecurity/trivy-db:2
```
[the remark this answers is not in the document]
Comment (FranceBe) [reply .github/workflows/aqua.yml]: Should be fixed :)

**Comment by FranceBe** [comment]
Hi @gdraynz ,
We changed node version, but I don't get where this CI check is coming from ? 
I don't see any other reference to node 12 in the repo, what am I missing ? :( 

Capture d’écran 2024-11-26 à 12 40 10

**Review by FranceBe — APPROVED** [review]

**Review by gdraynz — APPROVED** [review]
I will swap the required check when the PR is approved

**Comment by gdraynz** [comment]
@FranceBe Done, sorry I thought I did earlier :grin:

**Pull request merged**